components/dashboard/src/service/service.tsx

```diff
@@ -56,7 +58,67 @@ export function getGitpodService(): GitpodService {
         const service = _gp.gitpodService || (_gp.gitpodService = require("./service-mock").gitpodServiceMock);
         return service;
     }
-    const service = _gp.gitpodService || (_gp.gitpodService = createGitpodService());
+    let service = _gp.gitpodService;
+    if (!service) {
+        let user: any;
```

Comment:
nit: Would be cool to encapsulate the test code here a bit more. But as along as we keep track of removing it later it's fine with me. :+1: